Fix redefinition of htonll and consequent build failure on macOS.
https://github.com/memcached/memcached/issues/598
https://github.com/memcached/memcached/commit/95c67710aaf5cfe188d94b510faef8c66d6f5604
--- util.h.orig
+++ util.h
@@ -1,3 +1,5 @@
+#include "config.h"
+
 /* fast-enough functions for uriencoding strings. */
 void uriencode_init(void);
 bool uriencode(const char *src, char *dst, const size_t srclen, const size_t dstlen);
